presto-docs/src/main/sphinx/release/release-342.md

```diff
@@ -0,0 +1,52 @@
+# Release 342 (XX Sep 2020)
+
+## General Changes
+
+* Improve performance of queries that use decimal type. ({issue}`4886`)
+* Improve of queries involving `IN` with subqueries by extending support for dynamic
+  filtering and dynamic partition pruning. ({issue}`5017`)
+* Improve dynamic partition pruning and query performance by reducing latency of dynamic filters collection. ({issue}`4988`)
+* Improve join performance when cost-based optimizer has missing or inaccurate stats. ({issue}`5141`)
+* Enable connectors to wait for dynamic filters derived from replicated join before generating splits. ({issue}`4685`)
+* Fix query failure when lambda expression references a table column containing a dot. ({issue}`5087`)
+
+## Atop Connector Changes
+
+* Fix incorrect query results when query contains predicates on `start_time` or `end_time` column. ({issue}`5125`)
+
+## Hive Connector Changes
+
+* Allow specifying what happens if data is inserted into existing Hive partition. 
+  This can be done using `hive.insert-existing-partitions-behavior` config property. ({issue}`4999`)
+* Add property (`hive.dynamic-filtering-probe-blocking-timeout`) for delaying table scans
+  until dynamic partition pruning can be performed more efficiently. ({issue}`4991`)
+* Disable matching the existing user and group of the table or partition when creating new files on HDFS.
+  The functionality was added in 341 and is now disabled by default. You can enable it with `hive.fs.new-file-inherit-ownership`
+  configuration property. ({issue}`5187`)
+    
+## Kafka Connector Changes
+
+* Expose message headers as a `_headers` column of `map(VARCHAR, array(VARBINARY))` type. ({issue}`4462`)
+* Add write support for `TIME`, `TIME WITH TIME ZONE`, `TIMESTAMP` and `TIMESTAMP WITH TIME ZONE` 
+  for Kafka connector when JSON encoder is in use. ({issue}`4743`)
+  
+## MySQL Connector Changes
+
+* Improve performance of `INSERT` statement when MySQL instance isn't running in GTID mode. ({issue}`4995`)
+
+## PostgreSQL Connector Changes
+
+* Extend type mapping to support variadic `TIMESTAMP` and `TIMESTAMP WITH ZONE` types. ({issue}`5124`, {issue}`5105`)
+
+## SQL Server Connector Changes
+
+* Fix failure when inserting `NULL` to `VARBINARY` column. ({issue}`4846`)
+* Improve performance of aggregation queries by computing aggregations within SQL Server database.
+  Currently, the following aggregate functions are eligible for pushdown:
+  ``count``,  ``min``, ``max``, ``sum`` and ``avg``. ({issue}`4139`)
+
+## SPI Changes
+
+* Make dynamic filter futures resilient to cancellation. ({issue}`5099`)
```

Comment:
This is a change in engine's behavior, but is it really a SPI change?
@sopel39

Reply:
It would be too technical for general change. It's mostly relevant to SPI

Reply:
Is it relevant to a connector's implementor?
if not, maybe we just remove it from here? @sopel39 ?

Reply:
I haven't seen people complaining :) about this issue. OK to remove it